src/components/map/models/openlayers.ts

```diff
@@ -20,3 +21,29 @@ type Polygon = IContour<'Polygon', Coordinate[][]>;
 type MultiPolygon = IContour<'MultiPolygon', Coordinate[][][]>;
 
 export type Contour = Polygon | MultiPolygon;
+
+export interface Aanduidingsobject {
```

Comment:
kunnen we deze niet wat anders noemen? er  verwaaring zijn met de aanduidingsobjecten structuur die van de invetaris komt